index.bs

```diff
@@ -3260,6 +3315,11 @@ partial dictionary MLOpSupportLimits {
     <td>[=/any data type|any=]</td>
     <td>[=/any rank|N=]</td>
   </tr>
+  <tr>
+    <td>*output*</td>
+    <td>[=/same type as|same as=] {{input}}</td>
+    <td>{{newShape}}'s [=list/size=]</td>
```

Comment:
```suggestion
    <td>maximum of {{input}}'s [=MLOperand/rank=] and {{newShape}}'s [=list/size=]</td>
```
Because unidirectional broadcasting.

Reply:
This seems wrong - if _unidirectional_, isn't it just broadcasting input's shape up to newShape ? And in the algorithm for unidirectional broadcasting, sizeTo doesn't depend on shape from.

Reply:
You're right. Resolving.